src/main/java/frc/team2412/robot/commands/shooter/ShooterTargetCommand.java

```diff
@@ -42,8 +48,30 @@ public void execute() {
             shooter.setHoodAngle(shooterData.getAngle());
             shooter.setFlywheelRPM(shooterData.getRPM());
         }
-        turretAngle = turretEnable.getAsBoolean() ? turretAngle + localizer.getYaw() : 0;
-        shooter.setTurretAngle(turretAngle + localizer.yawAdjustment());
+        if (!turretEnable.getAsBoolean())
```

Comment:
I think the wrapping logic should go in `ShooterSubsystem` since that would apply to all uses of turret.

Reply:
For this level of complexity, I think having the wrapping logic in the command makes sense as it's much more of a 'control system' that fuses information from multiple sources (like here with the drivebase, camera, and current turret position). Much of this could also be refactored out into a utility helper class so the command is just something like `shooter.setTurretAngle(getWrappedAndYawAdjusted(currentTurreetAngle, localizer))`